server/player.lua

```diff
@@ -216,183 +230,231 @@ QBCore.Player.CreatePlayer = function(PlayerData)
 			end
+
 			TriggerClientEvent("hud:client:OnMoneyChange", self.PlayerData.source, moneytype, amount, false)
 			return true
 		end
+
 		return false
 	end
 
 	self.Functions.RemoveMoney = function(moneytype, amount, reason)
-		reason = reason ~= nil and reason or "unkown"
+		reason = reason ~= nil and reason or "unknown"
 		local moneytype = moneytype:lower()
 		local amount = tonumber(amount)
-		if amount < 0 then return end
+
+		if amount < 0 then
+			return
+		end
+
 		if self.PlayerData.money[moneytype] ~= nil then
 			for _, mtype in pairs(QBCore.Config.Money.DontAllowMinus) do
 				if mtype == moneytype then
 					if self.PlayerData.money[moneytype] - amount < 0 then return false end
 				end
 			end
+
 			self.PlayerData.money[moneytype] = self.PlayerData.money[moneytype] - amount
 			self.Functions.UpdatePlayerData()
+
 			if amount > 100000 then
-				TriggerEvent("qb-log:server:CreateLog", "playermoney", "RemoveMoney", "red", "**"..GetPlayerName(self.PlayerData.source) .. " (citizenid: "..self.PlayerData.citizenid.." | id: "..self.PlayerData.source..")** $"..amount .. " ("..moneytype..") removed, new "..moneytype.." balance: "..self.PlayerData.money[moneytype], true)
+				TriggerEvent("qb-log:server:CreateLog", "playermoney", "RemoveMoney", "red", "**" .. GetPlayerName(self.PlayerData.source) .. " (citizenid: " .. self.PlayerData.citizenid .. " | id: " .. self.PlayerData.source .. ")** $" .. amount .. " (" .. moneytype .. ") removed, new " .. moneytype .. " balance: " .. self.PlayerData.money[moneytype], true)
 			else
-				TriggerEvent("qb-log:server:CreateLog", "playermoney", "RemoveMoney", "red", "**"..GetPlayerName(self.PlayerData.source) .. " (citizenid: "..self.PlayerData.citizenid.." | id: "..self.PlayerData.source..")** $"..amount .. " ("..moneytype..") removed, new "..moneytype.." balance: "..self.PlayerData.money[moneytype])
+				TriggerEvent("qb-log:server:CreateLog", "playermoney", "RemoveMoney", "red", "**" .. GetPlayerName(self.PlayerData.source) .. " (citizenid: " .. self.PlayerData.citizenid .. " | id: " .. self.PlayerData.source .. ")** $" .. amount .. " (" .. moneytype .. ") removed, new " .. moneytype .. " balance: " .. self.PlayerData.money[moneytype])
 			end
+
 			TriggerClientEvent("hud:client:OnMoneyChange", self.PlayerData.source, moneytype, amount, true)
+
 			if moneytype == "bank" then
 				TriggerClientEvent('qb-phone:client:RemoveBankMoney', self.PlayerData.source, amount)
 			end
 			return true
 		end
+
 		return false
 	end
 
 	self.Functions.SetMoney = function(moneytype, amount, reason)
-		reason = reason ~= nil and reason or "unkown"
+		reason = reason ~= nil and reason or "unknown"
 		local moneytype = moneytype:lower()
 		local amount = tonumber(amount)
-		if amount < 0 then return end
+
+		if amount < 0 then
+			return
+		end
```

Comment:
Again, method that was used was way better then this one. This just add more lines, please make the changes.